Pyrado/pyrado/algorithms/step_based/value_based.py

```diff
@@ -200,7 +217,16 @@ def step(self, snapshot_mode: str, meta_info: dict = None):
     def update(self):
         raise NotImplementedError
 
-    def reset(self, seed: Optional[int] = None):
+    def reset(self, seed: Optional[int] = None, fill_memory_with_dummy_policy: bool = False):
```

Comment:
do we now still need `fill_memory_with_dummy_policy` since we got the new `self._use_trained_policy_for_refill`?

Reply:
if not, we also don't need a doc string for this function.

Reply:
Removed in e487cac5a8ef7697a54e54c1b20e09ce3831cbbb